bootstrap/scripts/buildAll.js

```diff
@@ -13,10 +13,11 @@ const $ = (cmd) => {
 
 console.log("building all...");
 $(`npx lerna run build --scope @dendronhq/common-all`);
-$(`npx lerna run build --scope @dendronhq/common-server`);
-$(`npx lerna run build --scope @dendronhq/dendron-viz`);
-$(`npx lerna run build --scope @dendronhq/engine-server`);
-$(`npx lerna run build --scope @dendronhq/pods-core`);
+$(`npx lerna run build --scope @dendronhq/unified`);
+$(`npx lerna run build --scope @dendronhq/common-server `);
```

Comment:
since common-server doesn't have a dependency on unified, can we run it in parallel with `unified`